docs-web/releases.md

```diff
@@ -1,5 +1,96 @@
 # Releases
 
+### NGINX Ingress Controller 1.10.0
```

Comment:
For consistency with 1.9.0:

```suggestion
## NGINX Ingress Controller 1.10.0
```

However, I don't see any difference in resulting page: https://deploy-preview-1325--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/releases/